docker/ubuntu-base/Dockerfile

```diff
@@ -1,12 +1,12 @@
 # Build remco from specific commit
 ##################################
-FROM golang:1.22.2 as remco
+FROM golang:1.24.1 as remco
 
 # REMCO_VERSION can be a release version "v0.XX.X", commit id, tag or branch name.
 
 # 2024-03-15: As of this writing, the latest remco version (v0.12.4) doesn't include the fixes for CVE-2023-44487
```

Comment:
The comment here can be updated.  The linked build is version 0.12.5 which does include the fixes referenced there and the new CVE fixes.